src/core/parse.js

```diff
@@ -35,7 +35,9 @@ export let parse = (obj, paren, wrapper) => {
                 blocks += key + '{' + parse(val, '', '') + '}';
             } else {
                 // Call the parse for this block
-                blocks += parse(val, next, next == paren ? key : wrapper || '');
+                next.split(/,/g).some((sib) => {
+                    blocks += parse(val, sib, sib == paren ? key : wrapper || '');
+                });
```

Comment:
🎉  awesome! 😃 

Was playing around to bring this down, and locally this version
```suggestion
                blocks += next.replace(/([^,])+/g, (group) =>
                    parse(val, group, group == paren ? key : wrapper || '')
                );
```

Adds like `+12B`. What do you think?

Reply:
Tried out the suggestion but somehow the test fails. Need to check what's going on 👍